app/views/admin/suppliers/show.html.haml

```diff
@@ -2,6 +2,27 @@
   .govuk-grid-column-full
     %h1.govuk-heading-xl= @supplier.name
 
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h2.govuk-heading-m{:class => 'govuk-!-margin-top-7'}
+      Tasks
+
+    -if @tasks.present?
```

Comment:
Yep! I did a bit of a deep dive into this last week! Because in most cases Tasks has items in it that are then all used to render the table ```present``` results in one less DB call.

Was a result of this:

https://github.com/dxw/DataSubmissionServiceAPI/pull/219#discussion_r250239487

Reply:
Fair enough :+1: